templates/app/orchestrator.yaml

```diff
@@ -30,7 +30,7 @@ objects:
               command:
               - pidof
               - MIQ Server
-            initialDelaySeconds: 480
+            initialDelaySeconds: ${{ORCHESTRATOR_INITIAL_DELAY}}
```

Comment:
The failing test is a format check so it's asking for quotes in this case, but does that make sense as this wasn't a string before? Also why the double curly braces?

Reply:
The template parameters don't play nice with non-string types. On my system, the double brackets worked to interpret it as an int and not a string.